app/uk/gov/hmrc/cataloguefrontend/whatsrunningwhere/WhatsRunningWhereController.scala

```diff
@@ -57,14 +57,32 @@ class WhatsRunningWhereController @Inject() (
   def releases(showDiff: Boolean): Action[AnyContent] =
     BasicAuthAction.async { implicit request =>
       for {
-        form <- Future.successful(WhatsRunningWhereFilter.form.bindFromRequest)
-        profile             = profileFrom(form)
-        selectedProfileType = form.fold(_ => None, _.profileType).getOrElse(ProfileType.Team)
+        form                 <- Future.successful(WhatsRunningWhereFilter.form.bindFromRequest)
+        profile               = profileFrom(form)
+        selectedProfileType   = form.fold(_ => None, _.profileType).getOrElse(ProfileType.Team)
         (releases, profiles) <- (service.releasesForProfile(profile).map(_.sortBy(_.applicationName.asString)), service.profiles).mapN((r, p) => (r, p))
-        environments = distinctEnvironments(releases)
-        profileNames = profiles.filter(_.profileType == selectedProfileType).map(_.profileName).sorted
+        environments          = distinctEnvironments(releases)
+        profileNames          = profiles.filter(_.profileType == selectedProfileType).map(_.profileName).sorted
       } yield Ok(page(environments, releases, selectedProfileType, profileNames, form, showDiff))
     }
+
+  def releasesConfig(showMemoryUse: Boolean): Action[AnyContent] = {
+    //The threshold of memory across instances and slots, for which the RGBA alpha value will be at its maximum.
+    //Any slotsAndInstancesToMemory values above this will be bounded to this figure.
+    val maxMemory = 32768.0
```

Comment:
AMBER: I would read this from config and put the value into application.conf, so it can be overridden without a code change

Reply:
Added a class to the service, not sure it should have it's own config file just for a single function?